app/jobs/send_match_sms_job.rb

```diff
@@ -12,20 +12,44 @@ def perform(match_id)
     match.set_expiration!
 
     begin
-      client = Twilio::REST::Client.new
-      message = client.messages.create(
-        from: "COVIDLISTE",
-        to: match.user.phone_number,
-        body: "Bonne nouvelle, une dose de vaccin vient de se libérer près de chez vous. Réservez-la vite sur : #{cta_url(match)}"
-      )
-      match.update(sms_sent_at: Time.now.utc, sms_provider: :twilio, sms_provider_id: message.sid)
-    rescue Twilio::REST::TwilioError => e
-      Rails.logger.info("[SendMatchSmsJob] #{e.message}")
+      provider = Flipper.enabled?(:sendinblue, match) ? "sendinblue" : "twilio"
+      sms_provider_id = send_with_provider(match, provider)
+      match.update(sms_sent_at: Time.now.utc, sms_provider: provider, sms_provider_id: sms_provider_id)
+    rescue => e
```

Comment:
At the very least :
```suggestion
    rescue StandardError => e
```

But if you know which errors are most likely to happen, better rescue those errors directly